js/element.js

```diff
@@ -1109,8 +1126,7 @@ export let fullScreen = (element = req("HTMLElement", "element")) => {
         if (element.requestFullScreen) return element.requestFullScreen();
 	else if (element.mozRequestFullScreen) return element.mozRequestFullScreen();
 	else if (element.webkitRequestFullScreen) return element.webkitRequestFullScreen();
-	else return new Error("Fullscreen failed")
-	);
+	else return new Error("Fullscreen failed");
```

Comment:
I feel like we should make this have brackets for readability, e.g:
```js
if (element.requestFullScreen){
  return element.requestFullScreen();
} else if (/* ... */){
   //...
}
```

Reply:
Sure thing, I won't add it in this pr tho, that was just a syntax error there